GNUmakefile

```diff
@@ -79,7 +79,7 @@ presubmit: ## Format go files and imports.
 
 .PHONY: mockery
 mockery: $(mockery) ## Install mockery.
-	go install github.com/vektra/mockery/v2@v2.12.2
+	go install github.com/vektra/mockery/v2@v2.13.0-beta.1
```

Comment:
Note: installing via `go install` is not recommended and may soon stop working https://github.com/vektra/mockery#go-install